integration_tests/e2e/add-steps.cy.ts

```diff
@@ -81,13 +81,21 @@ describe('Add Steps', () => {
       addStep.addStepAutocompleteText(2, secondStep.description)
       addStep.selectStepActor(2, secondStep.actor)
 
+      addStep.AddAnotherStepPressingEnter(2)
```

Comment:
Extended a current test to show that pressing enter now has the same functionality as selecting the "add another step" button